topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -275,7 +275,7 @@ We have codified all of the dependencies you will need into a YAML file that `an
 >    @@ -0,0 +1,13 @@
 >    +# Galaxy, Postgres, Nginx
 >    +- src: galaxyproject.galaxy
->    +  version: 0.10.14
+>    +  version: 0.10.15
```

Comment:
Is there a necessary update in 0.10.15 that fixes your fatal error?

Reply:
OK (as part of the upcoming commit).

Reply:
OK (part of the upcoming commit)